orienteer-core/src/main/java/org/orienteer/core/boot/loader/util/InitUtils.java

```diff
@@ -153,4 +153,14 @@ public String getOrienteerVersion() {
         String version = PROPERTIES.getProperty(ORIENTEER_VERSION);
         return Strings.isEmpty(version)?OrienteerWebApplication.class.getPackage().getImplementationVersion():version;
     }
+
+    private String getDefaultLibsFolder() {
+        String appHome = StartupPropertiesLoader.getAppHome();
```

Comment:
Lets Adjust getAppHome() - so it always return something. If no outside property - it should just return `.` as reference to current working dir.

Reply:
Adjusted